src/btree/bt_vrfy.c

```diff
@@ -276,7 +276,8 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             /* Verify the tree. */
             WT_WITH_PAGE_INDEX(
               session, ret = __verify_tree(session, &btree->root, &addr_unpack, vs));
-
+/* FIXME-WT-10927 */
```

Comment:
Is it the right FIXME ticket?

Reply:
Fixed it here: [f44a45d](https://github.com/wiredtiger/wiredtiger/pull/9651/commits/f44a45d3c444bc4f0803a7ce19179b3e59d95826)